### PR TITLE
QMessageBox/setWindowTitle: fix headers titles: capitalization, syntax + try to improve entries.

### PR DESCRIPTION
**Pull request by un1versal**
This pretty much squashes all the remainder dialog titles that were inconsistent, vague and incomplete or confusing to some translators.

Finally honor the dialog titles tips from https://github.com/kvirc/KVIrc/blob/master/doc/ui-style.tips.txt   like adding **\- KVIrc** to the title

Probably could be improved slightly hence the PR to gain valuable feedback.

### CODE REVIEW

**Comment by un1versal** [comment]
@staticfox you feel like rummaging here?

File: src/kvirc/kernel/KviApplication.cpp [hunk 1]
```diff
@@ -1023,11 +1023,11 @@ void KviApplication::checkSuggestRestoreDefaultScript()
...
-		QMessageBox::question(0,__tr2qs("Installation problems?"),
-			__tr2qs("<b>Oops...</b><br><br>" \
-				"<b>There are some reasons that make me think that your KVIrc installation is incomplete.</b><br><br>" \
-				"I might be wrong, but you seem to miss some of the features that the default KVIrc script provides." \
-				"This may happen because you have upgraded to an unstable Git version, " \
+		QMessageBox::question(0,__tr2qs("Detected Installation Problems - KVIrc"),
+			__tr2qs("<b>Oops!</b><br><br>" \
+				"<b>There are some reasons that make me think your KVIrc installation is incomplete.</b><br><br>" \
+				"I might be wrong, but you seem to be missing some of the features that the default KVIrc script provides." \
+				"This may happen because you have upgraded to a unstable Git version, " \
```
Comment (staticfox) [inline src/kvirc/kernel/KviApplication.cpp]: `an`

File: src/modules/sharedfileswindow/SharedFilesWindow.cpp [hunk 2]
```diff
@@ -174,7 +174,7 @@ void SharedFileEditDialog::okClicked()
 	QFileInfo f(szPath);
 	if(!(f.exists() && f.isFile() && f.isReadable()))
 	{
-		QMessageBox::warning(this,__tr2qs_ctx("Can't open the file","sharedfileswindow"),
+		QMessageBox::warning(this,__tr2qs_ctx("Can't Open the File - KVIrc","sharedfileswindow"),
```
Comment (staticfox) [inline src/modules/sharedfileswindow/SharedFilesWindow.cpp]: Hmm, I'm not sure about contractions within titles?

**Comment by staticfox** [comment]
Skimmed through it, should we be using contractions within titles/window titles? Let alone at all?

File: src/modules/sharedfileswindow/SharedFilesWindow.cpp (hunk 2 above)
Comment (un1versal) [inline src/modules/sharedfileswindow/SharedFilesWindow.cpp]: Probably not, changed this one at least. Now reads "Error Opening File - KVIrc"

**Comment by un1versal** [comment]
Closing to avoid IRC sync noise, will re-open later.

Still OK for review, please submit feedback/suggestions.

**Pull request closed without merging**